x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -222,14 +222,14 @@ export class FleetPlugin
     registerEncryptedSavedObjects(deps.encryptedSavedObjects);
 
     // Register feature
-    // TODO: Flesh out privileges
     if (deps.features) {
       deps.features.registerKibanaFeature({
-        id: PLUGIN_ID,
-        name: 'Fleet and Integrations',
+        id: `fleetv2`,
+        name: 'Fleet',
         category: DEFAULT_APP_CATEGORIES.management,
-        app: [PLUGIN_ID, INTEGRATIONS_PLUGIN_ID, 'kibana'],
+        app: [PLUGIN_ID, INTEGRATIONS_PLUGIN_ID],
```

Comment:
We shouldn't grant access to the Integrations app as part of Fleet privileges at all anymore. That was only from an earlier proposal where there was going to be an implicit connection between the two, which we decided against. Let's remove `INTEGRATIONS_PLUGIN_ID` from the `app` array on all of the `fleetv2` privileges. It should only be in the `fleet` (Integrations) privilege.

Reply:
You're right, this is a leftover of the previous implementation. I'll remove it :)